From dda3eeb61ab5134da02ea916d7be10434dc18b94 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Thu, 7 Jul 2022 10:32:07 +0200
Subject: [PATCH 1/1] etc: install system unit with without executable bit

All systemd unit files are expected to be not executable, so install
with correct permissions.
---
 etc/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/Makefile b/etc/Makefile
index 6a03c32..c47c180 100644
--- a/etc/Makefile
+++ b/etc/Makefile
@@ -81,10 +81,10 @@ install_systemd_service_files: $(DESTDIR)$(systemddir)/system $(SYSTEMD_DEST_SVC
 install_systemd_generator_files: $(DESTDIR)$(systemddir)/system-generators $(SYSTEMD_DEST_GEN_FILES)
 
 $(SYSTEMD_DEST_SVC_FILES): $(DESTDIR)$(systemddir)/system/%: systemd/%
-	$(INSTALL) $? $@
+	$(INSTALL) -m 644 $? $@
 
 $(SYSTEMD_DEST_GEN_FILES): $(DESTDIR)$(systemddir)/system-generators/%: systemd/%
-	$(INSTALL) $? $@
+	$(INSTALL) -m 755 $? $@
 
 install_iname: $(DESTDIR)$(HOMEDIR) $(ISCSI_INAME)
 	if [ ! -f $(INAME_DEST_FILE) ]; then \
-- 
2.37.0

